lib/matplotlib/pyplot.py

```diff
@@ -364,9 +364,40 @@ def isinteractive():
     return matplotlib.is_interactive()
 
 
+class _ioff:
+    def __init__(self):
```

Comment:
Should get some explanatory docstring. Something along the lines of

~~~
    """
    Context manager for `.ioff`.
    
    The state is changed in ``__init__()`` instead of ``__enter__()``. The
    latter is a no-op. This allows using `.ioff` both as a function and
    as a context.
    """
~~~